main.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:the_boxes/views/Rect4.dart';
+import 'package:the_boxes/views/Rect400.dart';
+import 'package:the_boxes/views/Rect42.dart';
+import 'package:the_boxes/views/rect40.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        title: 'Rectangles',
+        theme: ThemeData(
+          primarySwatch: Colors.blue,
+        ),
+        home: const MyHomePage());
+  }
+}
+
+class MyHomePage extends StatefulWidget {
```

Comment:
why stateful?

Reply:
no particular reason, I used the boilerplate code to write the homepage, I have changed this in the second pull request.